app/assets/javascripts/comments.coffee

```diff
@@ -0,0 +1,18 @@
+class Comment
+  constructor: ->
+    @bindEvents()
+
+  bindEvents: =>
+    $(document).on "ajax:success", ".comment-destroy", @destroyComment
```

Comment:
it better to use `data-beheivour` attributes instead of css classes https://flatstack.atlassian.net/wiki/pages/viewpage.action?pageId=13336599